modules/test/playwright/pages/layout-admin-web/WidgetPagePage.ts

```diff
@@ -106,6 +107,17 @@ export class WidgetPagePage {
 		}
 	}
 
+	async openLookAndFeel() {
```

Comment:
I think this should go directly on the test. You are checking the Asset Publisher options, and `WidgetPagePage` should not reference to particular widgets, unless you find a generic way to access widget options